llpc/test/shaderdb/relocatable_shaders/PipelineVsFs_ImmutableSampler.pipe

```diff
@@ -0,0 +1,77 @@
+// This test case checks that relocatable shaders are disabled if immutable shaders
+// are present (descriptorRangeValue entry exists).
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-NOT: Building pipeline with relocatable shader elf
```

Comment:
This looks like a good candidate for the matching against the new warning: https://github.com/GPUOpen-Drivers/llpc/pull/1210

Reply:
Done.